featuretools/entityset/entity.py

```diff
@@ -96,7 +96,7 @@ def __repr__(self):
     @property
     def shape(self):
         '''Shape of the entity's dataframe'''
-        return self.df.shape
+        return (len(self.df), self.df.shape[1])
```

Comment:
Looks like this is needed so the plotting tests pass. I am thinking we might want to keep this as `self.df.shape` and xfail the tests that fail with Dask - otherwise if we keep the `len()` call and someone tries to print an Entity that is a Dask dataframe a compute will be triggered and it could be a slow running process depending on the size of the df.